### PR TITLE
Reduce number of tabs in observability pack details

### PR DESCRIPTION
**Pull request by LizBaker**
## Description

This removes the requirements and nerdpacks tabs from the observability pack details page, and mocks the requirements info in the right rail. I also adjusted some mobile styles to avoid a scroll bar on the tabs as much as possible

## Related Issue(s) / Ticket(s)

Closes https://github.com/newrelic/developer-website/issues/1434

## Screenshot(s)
Screen Shot 2021-07-08 at 2 19 43 PM
Screen Shot 2021-07-08 at 2 19 31 PM

### CODE REVIEW

File: src/components/Tabs/Bar.js [hunk 1]
```diff
@@ -58,6 +58,7 @@ const Bar = ({ children, className }) => {
       css={css`
         display: flex;
         width: 100%;
+        border-bottom: 1px solid var(--color-neutrals-300);
```
Comment (aswanson-nr) [inline src/components/Tabs/Bar.js:61]: This color is very bright in dark mode, maybe we could use something a little closer to the left rail separator.
Screen Shot 2021-07-08 at 3 59 28 PM

Comment (zstix) [inline src/components/Tabs/Bar.js:61]: Can we also supply a color for the dark theme? I think we were using `var(--color-dark-300)` before.

**Review by zstix — CHANGES_REQUESTED** [review]
Two 'lil things to update, but this looks great otherwise!

**Review by aswanson-nr — DISMISSED** [review]
The border-bottom for the tabs looks a bit bright on darkmode, but it was like that before this change so feel free to ignore my comment

**Comment by zstix** [comment]
I also think that this removed our mobile dropdown styles. Was that intentional?

Screen Shot 2021-07-08 at 13 02 42

File: src/components/Tabs/Bar.js (hunk 1 above)
Comment (LizBaker) [reply src/components/Tabs/Bar.js]: good catch, i'm trying out --divider-color which is also used by the right rail, do you think it's too subtle? also realized I'd lost the button underline when selected so I added that back

**Comment by LizBaker** [comment]
@zstix we didn't have great mobile styles for the tabs imo, it was a scroll bar previously which I found awkward

**Comment by zstix** [comment]
@LizBaker I think we had a browser-based dropdown (thus: https://github.com/newrelic/developer-website/issues/1421)

**Review by zstix — APPROVED** [review]
Ignore the comment about the dropdown, I forgot we were doing it based on user-agent and not device-width 🤦‍♂️

**Comment by LizBaker** [comment]
@zstix ohhh gotcha, plus this just adds flex wrap past the 760 break point so i don't think it'll affect dropdown stuff when that's resolved 🤷‍♀️

**Pull request merged**